docs/tutorial.ipynb

```diff
@@ -0,0 +1,364 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SynDiffix Usage Tutorial\n",
+    "\n",
+    "This notebook demonstrates how to use __SynDiffix__, an open-source library for generating statistically-accurate\n",
+    "and strongly anonymous synthetic data from structured data.\n",
+    "\n",
+    "We'll go through the process of loading and inspecting a toy dataset, creating a synthetic dataset that mimics the original,\n",
+    "computing some statistical properties over the two datasets and comparing them, and, finally, how to improve accuracy when\n",
+    "analyzing synthetic data.\n",
+    "\n",
+    "### Setup\n",
+    "\n",
+    "The `syndiffix` package requires Python 3.10 or later. We can install it using `pip`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install syndiffix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll need a toy dataset to play with, so let's install the `scikit-learn` package in order to use one of their popular reference datasets:"
```

Comment:
Hmm, `scikit-learn` is a child dependency of ours. Does it work to import it directly?

Reply:
I guess it's better to keep it explicit in any case.

Reply:
Yes, that was my thinking as well.